src/js/player.js

```diff
@@ -4856,6 +4859,45 @@ class Player extends Component {
       this.previousLogLevel_ = undefined;
       this.debugEnabled_ = false;
     }
+
+  }
+
+  /**
+   * Set or get current playback rates.
+   * Takes an array and updates the playback rates menu with the new items.
+   * Pass in an empty array to hide the menu.
+   * Values other than arrays are ignored.
+   *
+   * @fires Player#rateschange
+   * @param {number[]} newRates
+   *                   The new rates that the playback rates menu should update to.
+   *                   An empty array will hide the menu
+   * @return {number[]} When used as a getter will return the current playback rates
+   */
+  playbackRates(newRates) {
+    if (newRates === undefined) {
+      return this.cache_.playbackRates;
+    }
+
+    // ignore any value that isn't an array
+    if (!Array.isArray(newRates)) {
+      return;
+    }
+
+    // ignore any arrays that don't only contain numbers
+    if (!newRates.every((rate) => typeof rate === 'number')) {
+      return;
+    }
+
+    this.cache_.playbackRates = newRates;
+
+    /**
+    * fires when the player language change
+    *
+    * @event Player#rateschange
+    * @type {EventTarget~Event}
+    */
+    this.trigger('rateschange');
```

Comment:
perhaps this should be `playbackratesupdated` as this doesn't signify that a new value was selected. It just shows that the list of values was updated. Also this comment is incorrect.

Reply:
Oops, forgot to update the jsdoc for it.
Given that `ratechange` is already a thing for when the playback rate changes, I was using that as a template for the new event name. Since the _rates changed_ as part of calling this method. I think this fits with the rest of the DOM naming of things. Maybe `playbackrateschange` to make it clear it's different from `ratechange`? Just the single extra `s` is hard to notice.

Reply:
`playbackrateschange` works for me.

Reply:
Updated.